corehq/apps/hqadmin/management/commands/check_services.py

```diff
@@ -0,0 +1,41 @@
+from collections import namedtuple
+from StringIO import StringIO
+from django.core.management import BaseCommand
+from corehq.blobs import get_blob_db
+
+
+ServiceStatus = namedtuple("ServiceStatus", "success msg")
+
+
+def test_blobdb():
+    """Save something to the blobdb and try reading it back."""
+    db = get_blob_db()
+    contents = "It takes Pluto 248 Earth years to complete one orbit!"
+    info = db.put(StringIO(contents))
+    with db.get(info.identifier) as fh:
+        res = fh.read()
+    db.delete(info.identifier)
+    if res == contents:
+        return ServiceStatus(True, "Successfully saved a file to the blobdb")
+    return ServiceStatus(False, "Did not successfully save a file to the blobdb")
```

Comment:
Maybe rephrase this to not include the word "successfully." Something like "Failed to save a file to the blobdb"